kodilla-sudoku/src/main/java/com/kodilla/sudoku/engine/SudokuGame.java

```diff
@@ -47,7 +47,7 @@ private void resolveSudoku() {
                 while (element.getValue() == SudokuElement.EMPTY) {
                     newElement.setValue(random.nextInt(BOARD_SIZE) + 1);
                     if (checker.check(newElement)) {
-                        board.addPlayerMove(newElement);
+                        element.setValue(element.getValue());
```

Comment:
na pewno tak? dla mnie to wywolanie nic nie zmienia: value jest ustawiane na dotychczasowe value